src/components/LoginComponent.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import "../styles/form.style.scss"
+import {Formik} from "formik"
+import * as yup from "yup"
+import { useNavigate } from 'react-router-dom'
+
+const mySchema = yup.object().shape({
+  Email: yup.string().required("Username is requires!"),
+  Password: yup.string().min(8, "login must be at least 8 chars long!")
+})
+
+function LoginComponent() {
+  const navigate = useNavigate()
+
+  function submitForm(values,action){
+    console.log(values)
+    navigate("/dashboard")
+  }
+
+  return (
+    <Formik 
+      initialValues={{
+      Email: "",
```

Comment:
You can name classes with uppercase, but variables are lower case, so:
email, password